### ISSUE TITLE
[TW-1578] Bash tab completion problems on first run

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1604]
_Renato Alves on 2015-03-17T20:13:38Z says:_ 

 As described by Ptolemarch on IRC
```
Ptolemarch | I've run into a problem with taskwarrior's bash-completion.
Ptolemarch | Namely, the completion script runs `task` itself, which assumes a .taskrc, and if one isn't there, prompts the user to create one.
Ptolemarch | The problem is that in the completion script, which is of course run as part of .bashrc, this prompt is invisible, meaning that my .bashrc hangs for me to type "yes"
Ptolemarch | The problem comes in when you install taskwarrior via a package manager that also installs the bash completion, and then you open a new terminal and get a blank screen with nothing but a blinking cursor.
Ptolemarch | So I'm fine now that I've figured it out, but I wanted to let you know.
```

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-03-17T20:13:38Z
Modified: 2015-03-20T00:35:11Z
```

**Comment by taskwarrior** [comment]
_Tomas Babej on 2015-03-20T00:34:31Z says:_ 

 Fixed with: 704eb1eab3a63fa5df6fcbf91a5eead78f30418d
